packages/main/src/FileUploader.js

```diff
@@ -264,7 +264,7 @@ class FileUploader extends UI5Element {
 	constructor() {
 		super();
 		if (this._canUseNativeFormSupport) {
-			this._internals = this.attachInternals();
+			this._internals = this.attachInternals && this.attachInternals();
 		}
```

Comment:
Remove the surrounding if statement. Let it be only:
```js
this._internals = this.attachInternals && this.attachInternals();
```

Reply:
Done